### ISSUE TITLE
pre-trained model doesn't show well on pose guided image generate

### ISSUE THREAD

**Issue by Aurora-Xuan** [RenYurui/Global-Flow-Local-Attention#81]
the picture generated successfully but the texture is poor. What werid is that 6 month ago I've tested the pre-trained model and the generated pictures are OK. Recently I try it again but the results are not good.
I only run "setup.bash" again, I haven't changed the other things. I aslo try to delete the cuda_extension files and rebuild the files but nothing changed. Any idea?

![before.jpg](http://ww1.sinaimg.cn/large/006sC94ply1gr3zz9ztrij3074074a9z.jpg)

![after.jpg](http://ww1.sinaimg.cn/large/006sC94ply1gr3zzo2593j3074074dfp.jpg)

**Comment by hcy1998** [comment]
hello,I found that it  does not exist of Script.PerceptualSimilarity.models in the repo.have you meet this trouble?

**Comment by Aurora-Xuan** [comment]
> hello,I found that it does not exist of Script.PerceptualSimilarity.models in the repo.have you meet this trouble?

I haven't test the metrics yet, sry for that.

**Comment by shilongshen** [comment]
> hello,I found that it does not exist of Script.PerceptualSimilarity.models in the repo.have you meet this trouble?

hi ! Did you solve this problem?

**Comment by hcy1998** [comment]
> > hello,I found that it does not exist of Script.PerceptualSimilarity.models in the repo.have you meet this trouble?
> 
> hi ! Did you solve this problem?

NO,The code still doesn't work

**Comment by KHao123** [comment]
The url of pretrained model is broken? Could you please provide me a url to download the model?
Thank you !

**Comment by hcy1998** [comment]
font{
        line-height: 1.6;
    }
    ul,ol{
        padding-left: 20px;
        list-style-position: inside;
    }


    
    All the  url of pretrained model is OK.Which do you want to download？Did you find the wrong download address？
    
        
            
        
    
    
     
    
        
                    
            
                
                    
                            
                                
                            
                            
                                921127494
                            
                    
                        
                            
                                    ***@***.***
                            
                        
                
            
        
        
    
    
        签名由
        网易邮箱大师
        定制
    
 

    On 7/8/2021 ***@***.***> wrote： 



The url of pretrained model is broken? Could you please provide me a url to download the model?
Thank you !

—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by RenYurui** [comment]
> The url of pretrained model is broken? Could you please provide me a url to download the model?
> Thank you !

Hi! 
The URL of Google Drive is available. Can you use these links？

**Comment by RenYurui** [comment]
> > > hello,I found that it does not exist of Script.PerceptualSimilarity.models in the repo.have you meet this trouble?
> > 
> > 
> > hi ! Did you solve this problem?
> 
> NO,The code still doesn't work

I will check and upload the file tomorrow.

**Comment by RenYurui** [comment]
> the picture generated successfully but the texture is poor. What werid is that 6 month ago I've tested the pre-trained model and the generated pictures are OK. Recently I try it again but the results are not good.
> I only run "setup.bash" again, I haven't changed the other things. I aslo try to delete the cuda_extension files and rebuild the files but nothing changed. Any idea?
> 
> ![before.jpg](https://camo.githubusercontent.com/44c01f64dc211dc62d76873e2bf2abc06917462820c98b8dc8ae74f600947ee6/687474703a2f2f7777312e73696e61696d672e636e2f6c617267652f30303673433934706c79316772337a7a397a7472696a3330373430373461397a2e6a7067)
> 
> ![after.jpg](https://camo.githubusercontent.com/3d0c92266a3b393623eaf3adac299b1bd24f91ca9141f02fd65251eb9bd1c1fb/687474703a2f2f7777312e73696e61696d672e636e2f6c617267652f30303673433934706c79316772337a7a6f323539336a333037343037346466702e6a7067)

Have you update your cuda or nvidia driver? 
Can you check the output of the local attention? Are they all zeros?

**Comment by Aurora-Xuan** [comment]
> > the picture generated successfully but the texture is poor. What werid is that 6 month ago I've tested the pre-trained model and the generated pictures are OK. Recently I try it again but the results are not good.
> > I only run "setup.bash" again, I haven't changed the other things. I aslo try to delete the cuda_extension files and rebuild the files but nothing changed. Any idea?
> > ![before.jpg](https://camo.githubusercontent.com/44c01f64dc211dc62d76873e2bf2abc06917462820c98b8dc8ae74f600947ee6/687474703a2f2f7777312e73696e61696d672e636e2f6c617267652f30303673433934706c79316772337a7a397a7472696a3330373430373461397a2e6a7067)
> > ![after.jpg](https://camo.githubusercontent.com/3d0c92266a3b393623eaf3adac299b1bd24f91ca9141f02fd65251eb9bd1c1fb/687474703a2f2f7777312e73696e61696d672e636e2f6c617267652f30303673433934706c79316772337a7a6f323539336a333037343037346466702e6a7067)
> 
> Have you update your cuda or nvidia driver?
> Can you check the output of the local attention? Are they all zeros?

Hi, thanks for replying!
1. we deleted the nvidia driver by accident 2 months ago but we reinstall it with the same version.
2. we check the local attention(attn_param in base_function.py), e.g. attn_param, block_source, block_target, result, they're all zeroes. is it ok for that?

**Comment by RenYurui** [comment]
> > > the picture generated successfully but the texture is poor. What werid is that 6 month ago I've tested the pre-trained model and the generated pictures are OK. Recently I try it again but the results are not good.
> > > I only run "setup.bash" again, I haven't changed the other things. I aslo try to delete the cuda_extension files and rebuild the files but nothing changed. Any idea?
> > > ![before.jpg](https://camo.githubusercontent.com/44c01f64dc211dc62d76873e2bf2abc06917462820c98b8dc8ae74f600947ee6/687474703a2f2f7777312e73696e61696d672e636e2f6c617267652f30303673433934706c79316772337a7a397a7472696a3330373430373461397a2e6a7067)
> > > ![after.jpg](https://camo.githubusercontent.com/3d0c92266a3b393623eaf3adac299b1bd24f91ca9141f02fd65251eb9bd1c1fb/687474703a2f2f7777312e73696e61696d672e636e2f6c617267652f30303673433934706c79316772337a7a6f323539336a333037343037346466702e6a7067)
> > 
> > 
> > Have you update your cuda or nvidia driver?
> > Can you check the output of the local attention? Are they all zeros?
> 
> Hi, thanks for replying!
> 
> 1. we deleted the nvidia driver by accident 2 months ago but we reinstall it with the same version.
> 2. we check the local attention(attn_param in base_function.py), e.g. attn_param, block_source, block_target, result, they're all zeroes. is it ok for that?

It is not correct. Try to remove all the building blocks with "rm -rf build dist *info" and reinstall.

**Comment by Aurora-Xuan** [comment]
> > > > the picture generated successfully but the texture is poor. What werid is that 6 month ago I've tested the pre-trained model and the generated pictures are OK. Recently I try it again but the results are not good.
> > > > I only run "setup.bash" again, I haven't changed the other things. I aslo try to delete the cuda_extension files and rebuild the files but nothing changed. Any idea?
> > > > ![before.jpg](https://camo.githubusercontent.com/44c01f64dc211dc62d76873e2bf2abc06917462820c98b8dc8ae74f600947ee6/687474703a2f2f7777312e73696e61696d672e636e2f6c617267652f30303673433934706c79316772337a7a397a7472696a3330373430373461397a2e6a7067)
> > > > ![after.jpg](https://camo.githubusercontent.com/3d0c92266a3b393623eaf3adac299b1bd24f91ca9141f02fd65251eb9bd1c1fb/687474703a2f2f7777312e73696e61696d672e636e2f6c617267652f30303673433934706c79316772337a7a6f323539336a333037343037346466702e6a7067)
> > > 
> > > 
> > > Have you update your cuda or nvidia driver?
> > > Can you check the output of the local attention? Are they all zeros?
> > 
> > 
> > Hi, thanks for replying!
> > 
> > 1. we deleted the nvidia driver by accident 2 months ago but we reinstall it with the same version.
> > 2. we check the local attention(attn_param in base_function.py), e.g. attn_param, block_source, block_target, result, they're all zeroes. is it ok for that?
> 
> It is not correct. Try to remove all the building blocks with "rm -rf build dist *info" and reinstall.

Hey it works! Thank you so much.
btw, I wonder what's the purpose for input changed from 176*256 to 256*256. Does it has specific reason on training or something else?

**Comment by imbinwang** [comment]
@Aurora-Xuan Do you know the reason for changing input size? When I trace the code in base_dataset.py, I found the size of input image is changed from 750x1101 to 256x256. Because of not keeping image ratio, this results into a 'fat' input image for network. The generated image seems also 'fatter' than ground-truth, as following figure shows.
![f1](https://user-images.githubusercontent.com/8622373/126591521-d944b69c-4900-4e08-901c-d8751d5156ce.png)
